packages/create-app/index.js

```diff
@@ -31,6 +31,7 @@ const TEMPLATES = [
 const renameFiles = {
   _gitignore: '.gitignore'
 }
+const projectNameRE = /^[A-Za-z0-9_-]*$/
```

Comment:
This is totally incorrect. Please use something like [validate-npm-package-name](https://www.npmjs.com/package/validate-npm-package-name)

Vite depends on [resolve.exports](https://github.com/vitejs/vite/blob/main/packages/vite/package.json#L113) which wouldn't pass your regex, for example.